server/server.js

```diff
@@ -28,7 +28,9 @@ const PORT = config.server.port;
 // Simple function to color text in the console output
 function logColor(msg, color) {
     var colors = {
+		red: ['\x1b[31m', '\x1b[0m'],
```

Comment:
Purely from a pedantic perspective but it looks like the indenting here is a bit off, can you fix that and then I can merge 😄